dev-libs/xapian/xapian-1.2.25.recipe

```diff
@@ -74,12 +81,17 @@ BUILD_PREREQUIRES="
 	cmd:sed
 	"
 
-defineDebugInfoPackage xapian$secondaryArchSuffix \
-	"$libDir"/libxapian.so.$libVersion
+debugList=("$libDir"/libxapian.so.$libVersion)
+for i in `echo "$PROVIDES" | sed -n -e \
+		"s/^[\t]*cmd:\(.*\)$commandSuffix =.*/\1/p;" | sed -e "s/_/-/g"`; do
```

Comment:
I'm not really a fan of using non-builtin when dealing with metadata though, as you can't be sure that external cmds exist, but I think it's fine here.

Reply:
Yeah, I also try to avoid unusual blocs like this one, but here, as you already noticed, its fine because there are a lot of `cmd:` items, so duplicating them here would be worse than having this tiny block.